### ISSUE TITLE
UnicodeEncodeError: 'charmap' codec can't encode character  '\u22c5' in position 190: character maps to <undefined>

### ISSUE THREAD

**Issue by finalsnd** [cbm755/octsympy#428]
```
>> syms t
>> syms t real
>> f = 2*(1+t)+4*pi*sin(pi*t)
error: Python exception: UnicodeEncodeError: 'charmap' codec can't encode character
 '\u22c5' in position 190: character maps to <undefined>
    occurred while copying vars from Python
error: called from
    python_cmd at line 178 column 5
    mtimes at line 42 column 5
```

Am I doing something wrong?

**Comment by cbm755** [comment]
Yes this is https://savannah.gnu.org/bugs/?func=detailitem&item_id=46771

I would really like some help with this!  In particular if you can test PR #302...  Many people have reported this but I cannot test it myself.

**Comment by cbm755** [comment]
merged #302, fixed (I hope).
